src/Utils/BlocksWpQuery.php

```diff
@@ -38,13 +38,24 @@ public function __construct( $query = '' ) {
 	}
 
 	/**
-	 * Cache cached posts, if a cache exists.
+	 * Get cached posts, if a cache exists.
+	 *
+	 * A hash is generated using the array of query_vars. If doing custom queries via filters such as posts_where
+	 * (where the SQL query is manipulated directly) you can still ensure there is a unique hash by injecting custom
+	 * query vars via the parse_query filter. For example:
+	 *
+	 *      add_filter( 'parse_query', function( $wp ) {
```

Comment:
@mikejolley, I think you should rename `$wp` with `$query`.

Reply:
Agreed.